tools/monorepo-utils/src/code-freeze/commands/version-bump/index.ts

```diff
@@ -8,24 +8,20 @@ import simpleGit from 'simple-git';
  * Internal dependencies
  */
 import { Logger } from '../../../core/logger';
-import { sparseCheckoutRepoShallow } from '../../../core/git';
+import {
+	sparseCheckoutRepoShallow,
+	checkoutRemoteBranch,
+} from '../../../core/git';
 import { createPullRequest } from '../../../core/github/repo';
 import { getEnvVar } from '../../../core/environment';
 import { getMajorMinor } from '../../../core/version';
 import { bumpFiles } from './bump';
 import { validateArgs } from './lib/validate';
 import { Options } from './types';
 
-const genericErrorFunction = ( err ) => {
-	if ( err.git ) {
-		return err.git;
-	}
-	throw err;
-};
-
 export const versionBumpCommand = new Command( 'version-bump' )
 	.description( 'Bump versions ahead of new development cycle' )
-	.requiredOption( '-v, --version <version>', 'Version to bump to' )
+	.argument( '<version>', 'Version to bump to' )
```

Comment:
This works better as an argument rather than required option. I changed the corresponding action to match this change.